test-data/unit/fine-grained-dataclass.test

```diff
@@ -0,0 +1,25 @@
+[case replace]
```

Comment:
By convention all test cases names start with `test` so it should be `[case testReplace]`

Reply:
cd35951b603b10cd4a7597b8d8f52d3137000e36